proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -201,6 +201,8 @@ message Service {
   ServiceOptions service_options = 8;
   // implicitly_deprecated is true if its enclosing file is deprecated.
   bool implicitly_deprecated = 9;
+
+  repeated Option options = 10;
```

Comment:
I see that many of the other fields also don't have doc comments, including the `service_options` field. But it might be nice to add something to each of these to clarify their purpose, and also to move them so they're adjacent. Something like so:
```suggestion
  // implicitly_deprecated is true if its enclosing file is deprecated.
  bool implicitly_deprecated = 9;

  // A select sub-set of service options.
  ServiceOptions service_options = 8;
  // All options that are present on the service. This is a super-set of
  // service_options and uses a dynamic representation so it can also
  // accommodate custom options with arbitrary types.
  repeated Option options = 10;
```